src/lib.rs

```diff
@@ -79,7 +79,7 @@ macro_rules! rect {
     );
     (
         $left:tt $($left_unit:ident)?, $top:tt $($top_unit:ident)?,
-        $right:tt $($right_unit:ident)?, $bottom:tt $($bottom_unit:ident)?,
+        $right:tt $($right_unit:ident)?, $bottom:tt $($bottom_unit:ident)?
```

Comment:
Does this work? It makes it optional.
```suggestion
        $right:tt $($right_unit:ident)?, $bottom:tt $($bottom_unit:ident)? $(,)?
```

Reply:
Seems to work. I did a test with and without the comma in my code and cargo expand seems to show the correct code.